docs/configuration.md

```diff
@@ -100,30 +100,227 @@ default. To enable them, use `--pprof` and `--metrics` flags or
 
```

Comment:
To be more consistent, shouldn't we remove mention to a default value in Monitoring and metrics?
` Pprof and Prometheus are integrated into the gateway, but not enabled by
default.`